### ISSUE TITLE
Generate Graph from link connections between headers

### ISSUE THREAD

**Issue by falematte** [alphapapa/org-graph-view#8]
A lot of workflows, e.g. roam, use links instead of hierarchies between headers. Links are more general and non directional. I think that being able to generate a graph from org headers in the same file linking to each other would be really usefull

**Comment by alphapapa** [comment]
That could be useful.  It probably wouldn't be too difficult to do.

**Comment by bepolymathe** [comment]
Oh yes it would be great !

**Comment by notuntoward** [comment]
Yeah, non-directional, arbitrary links are useful.  In fact, I'd really like it if this made graph edges following [dedicated targets](https://www.gnu.org/software/emacs/manual/html_node/org/Internal-links.html).  I use them constantly in my org files -- hierarchy is a great organizing principle, but there's a lot it just can't express.
